augur/export_v2.py

```diff
@@ -1154,6 +1154,14 @@ def run(args):
                 f"Valid delimiters are: {args.metadata_delimiters!r}. "
                 "This can be changed with --metadata-delimiters."
             )
+        except ValueError as error:
+            if len(metadata_df.index) > len(set(metadata_df.index)):
+                raise AugurError(
+                    f"The following {metadata_df.index.name} values are duplicated in '{args.metadata}':\n"
+                    + "\n".join(sorted(metadata_df.index[metadata_df.index.duplicated()])))
```

Comment:
_non-blocking_

Small suggestion to include a "solution" to the error: either deduplicate the metadata or provide a different id column via `--metadata-id-columns`.

Reply:
Thanks -- added `You can fix this by either de-duplicating the metadata, or setting a different unique index column via --metadata-id-columns.` and force-pushed; will merge once tests pass.